optuna/storages/_rdb/storage.py

```diff
@@ -644,6 +645,11 @@ def _get_prepared_new_trial(
 
         return trial
 
+    @deprecated(
+        "3.0.0",
+        "4.0.0",
```

Comment:
As default, we set `removed_version` as the two later major version of `deprecated_version`.
```suggestion
        "5.0.0",
```

Reply:
Ok, Fixed.